assets/js/modules/analytics-4/index.js

```diff
@@ -169,19 +168,15 @@ export const registerWidgets = ( widgets ) => {
 			wrapWidget: false,
 			modules: [ 'analytics-4' ],
 			isActive: ( select ) => {
-				const isAnalyticsConnected =
-					select( CORE_MODULES ).isModuleConnected( 'analytics-4' );
-
-				/**
-				 * TODO: This widget should be shown only if the audience group
-				 * is set up for the current user. This should be fixed once
-				 * the audience settings become accessible without `analytics-4`
-				 * module being connected.
-				 * See: https://github.com/google/site-kit-wp/issues/8810 for
-				 * more details.
-				 */
+				const configuredAudiences =
+					select( CORE_USER ).getConfiguredAudiences();
+				const isAudienceSegmentationWidgetHidden =
+					select( CORE_USER ).isAudienceSegmentationWidgetHidden();
 
-				return ! isAnalyticsConnected;
```

Comment:
@zutigrm, I believe we should still check whether the analytics module is connected in addition to audience settings checks to determine whether we need to show this widget.

Reply:
@zutigrm That is incorrect. The widget `ConnectAnalyticsCTAWidget` should only be active when the Analytics module is not fully connected in addition to the current logic, as there's no point in showing this comment if the module is already connected :smile_cat:

Reply:
@kuasha420 It is a bit confusing, since inline comment mentioned that it should be shown only: `This widget should be shown only if the audience group is set up for the current user` once settings are accessible without relying on analytics being connected or not. Which IB also suggested, to replace this current check with new logic. 
Should we keep this check still?

Reply:
Check has been re-included
@kuasha420 @eugene-manuilov